docs/pages/guides/CssOverrides.example.jsx

```diff
@@ -46,6 +49,7 @@ function CssOverrides() {
         label="Light blue picker"
         value={selectedDate}
         onChange={handleDateChange}
+        shouldDisableDate={day => day.getDay() === 0}
```

Comment:
Umm 😐 
Why it was added?

Reply:
I added this to trigger the illustration of `dayDisabled` style. So the `dayDisabled` style is added to Sundays.